CONTRIBUTING.md

```diff
@@ -77,6 +77,17 @@ Coding Conventions
 Use tabs. If you want to align lines, use spaces. Any desired alignment should
 display fine at any tab-length display setting.
 
+Please lint your code using [clippy](https://github.com/rust-lang/rust-clippy).
```

Comment:
Maybe "Note that our CI enforces a few basic `clippy` lints - you can test locally if you use rustup, or just push to CI to test." or something like that, given that clippy is only available via rustup and we *dont* want to encourge people to use rustup when they otherwise wouldn't (as a few other things we use break, eg the bindings generation link-time-LTO, which is way more important than clippy)

Reply:
awesome, thanks for the context!